### ISSUE TITLE
Cannot subscribe to 'page' store on the server

### ISSUE THREAD

**Issue by frederikhors** [Ernxst/svelte-query-params#4]
### Describe the bug

I'm getting this error when I first use your package 0.2.0:

```
Error: Cannot subscribe to 'page' store on the server outside of a Svelte component, as it is bound to the current request via component context. This prevents state from leaking between users.For more information, see https://kit.svelte.dev/docs/state-management#avoid-shared-state-on-the-server
    at get_store (C:\project\node_modules\@sveltejs\kit\src\runtime\app\stores.js:89:9)
    at Object.subscribe (C:\project\node_modules\@sveltejs\kit\src\runtime\app\stores.js:35:37)
    at Module.subscribe_to_store (C:\project\node_modules\svelte\src\store\utils.js:22:22)
    at Module.get_store_value (C:\project\node_modules\svelte\src\store\index.js:218:2)
    at getServerUrl (C:\project\node_modules\svelte-query-params\dist\adapters\sveltekit\index.js:21:14)
    at getQueryParams (C:\project\node_modules\svelte-query-params\dist\index.svelte.js:91:33)
    at Module.createUseQueryParams (C:\project\node_modules\svelte-query-params\dist\index.svelte.js:84:20)
    at C:\project\src\lib\utils\app\helpers.ts:310:31
```

I'm using `export const ssr = false;`.

### Reproduction

-

### System Info

```shell
-
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/Ernxst/svelte-query-params/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Ernxst/svelte-query-params/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://github.com/Ernxst/svelte-query-params/blob/main/docs).
- [X] Check that there isn't [already an issue](https://github.com/Ernxst/svelte-query-params/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/Ernxst/svelte-query-params/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by frederikhors** [comment]
> Could you share a repo?

Not right now. But is a new Kit app with svlete 5.163 and all deps to the latest ones.

**Comment by Ernxst** [comment]
Does this only happen during dev, or after a build + preview also?

**Comment by frederikhors** [comment]
Dev

**Comment by Ernxst** [comment]
I see that error when ssr _is_ enabled. In any case, I'll take a look

**Comment by Ernxst** [comment]
Should be fixed by [`0.2.1`](https://github.com/Ernxst/svelte-query-params/commit/06cd15fa3a27f5e6c8b80ecded274fec351385ad) - contains breaking changes as mentioned in the [changelog](https://github.com/Ernxst/svelte-query-params/releases/tag/svelte-query-params%400.2.1)
